INN/umbrella-inndev
# 126 amplify child theme

## Changes

This pull request makes the following changes:

- creates `amplify` child theme:
    - stylesheet
    - Grunt
    - gitignore
    - LESS variables and common styles pulled in from https://github.com/INN/inn/blob/master/less/variables.less to match other INN sites' branding
- updates the README in `themes/` to refer to more themes present on this site (see https://github.com/INN/umbrella-inndev/issues/60 for why some of these child themes aren't present in this repo)

## Why



For #126, creating the child theme for the Amplify site.

## Testing/Questions

Features that this PR affects:

- amplify.inn.org, which is currently being bootstrapped
- docs in this repo


Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository? I'm setting this to "staging" because I created the branch based off of staging, and because we'll be deploying to staging. It [looks like the current master is ahead of the current staging branch](https://github.com/INN/umbrella-inndev/network), so merging this to master and deploying to prod shouldn't bring unplanned commits into the prod env.
- [ ] is there any other boilerplate we want to include here for #126?

Steps to test this PR:

1. check out this branch
2. in `/wp-admin/network/sites.php`, create a new site, within the inndev multisite, for amplify work on your local env.
3. find the "edit site" view for your new amplify site, and go to the "themes" tab. Enable the "Amplify" theme.
4. Go to your amplify site's dashboard, and activate the "Amplify" theme.
